bindings/Python/CMakeLists.txt

```diff
@@ -47,18 +47,34 @@ set_target_properties(adios2_py PROPERTIES
   COMPILE_PDB_OUTPUT_DIRECTORY ${CMAKE_PYTHON_OUTPUT_DIRECTORY}/adios2
 )
 
-string(REGEX REPLACE "[^/]+" ".." relative_base "${CMAKE_INSTALL_PYTHONDIR}/adios2")
 if(CMAKE_SYSTEM_NAME MATCHES "Linux")
-  set_target_properties(adios2_py PROPERTIES
-    INSTALL_RPATH "$ORIGIN/${relative_base}/${CMAKE_INSTALL_LIBDIR}"
-  )
+  if (ADIOS2_PIP_INSTALL)
+    set_target_properties(adios2_py PROPERTIES
+      INSTALL_RPATH "${CMAKE_INSTALL_RPATH}"
+    )
+  else()
+    string(REGEX REPLACE "[^/]+" ".." relative_base "${CMAKE_INSTALL_PYTHONDIR}/adios2")
+    set_target_properties(adios2_py PROPERTIES
+      INSTALL_RPATH "$ORIGIN/${relative_base}/${CMAKE_INSTALL_LIBDIR}"
+    )
+  endif()
 endif()
 
-install(TARGETS adios2_py
-  DESTINATION ${CMAKE_INSTALL_PYTHONDIR}/adios2
-  COMPONENT adios2_python-python
-)
-install(FILES ${CMAKE_PYTHON_OUTPUT_DIRECTORY}/adios2/__init__.py
-  DESTINATION ${CMAKE_INSTALL_PYTHONDIR}/adios2
-  COMPONENT adios2_python-python
+if (ADIOS2_PIP_INSTALL)
+  install(TARGETS adios2_py
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    COMPONENT adios2_python-python)
+  install(FILES ${CMAKE_PYTHON_OUTPUT_DIRECTORY}/adios2/__init__.py
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}
```

Comment:
Would it make sense to set CMAKE_INSTALL_PYTHONDIR <- CMAKE_INSTALL_LIBDIR

Reply:
That might make sense here, and thus remove the need for the if/else conditional.  However, `CMAKE_INSTALL_PYTHONDIR` is used [elsewhere](https://github.com/ornladios/ADIOS2/blob/master/cmake/ADIOSFunctions.cmake#L80-L91), and I think your suggestion would result in the conditional needing to be moved over there instead.  Or else we would need similar conditional in `DetectOptions.cmake`.

But let me know if you think I missed the point of your suggestion.

Reply:
hmm maybe that is ok, maybe for pip builds we would also want to install python source code in the libdir (in the build directory for tests). That would be more consistent.

Reply:
I don't think I understand your whole vision here.  But if it's still not how you want it, maybe we can discuss in person?

Reply:
sure lets sync maybe in 30mins?

Reply:
We agreed in our discussion that this is fine for now.